### ISSUE TITLE
YouTube Playlists

### ISSUE THREAD

**Issue by m2n037** [m2n037/awesome-mecheng#23]
Hello, guys, I have added all my saved playlists in YouTube in [this doc](https://github.com/m2n037/awesome-mecheng/blob/master/youtube-playlists.md). This needs to be segregated into appropriate sections. Please help and share your thoughts.

**Comment by Clockmaker** [comment]
Looks like a lot of work 😄
Skimming through it I saw a lot of strange topics like ML, opengl and even Gimp.
Maybe if you start a bit slower, focusing on 3-4 topics I can help.
I think that many visitors are undergrads so math, physics and programming are a good start.

Let me know and keep up the good work

**Comment by m2n037** [comment]
Yes lots of things are there according to my personal preference. I plan to keep the mechanical stuff here and gradually migrate others to another repo
